README.rst

```diff
@@ -22,6 +22,106 @@ For more options, see::
 
     $ make help
 
+Configure using thinkhazard_processing.yaml
+===========================================
+
+Keys in configuration file:
+
+sqlalchemy.url
+--------------
+
+Database connection parameters, example :
+
+.. code:: yaml
+
+    sqlalchemy.url: postgresql://www-data:www-data@localhost:5432/thinkhazard_processing
+
+data_path
+---------
+
+Path to main data folder, example :
```

Comment:
Maybe mention that it should be a dedicated partition, to prevent side-effects if/when data is huge.